com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties

```diff
@@ -484,7 +484,7 @@ msgCanceled=Canceled
 msgCfmClearALL=Do you want to clear all selections?\nHowever, Added SQLs are not deleted.
 msgCfmDeleteSQL=Do you want to delete selected SQL?
 msgCfmErrorSave=Error occurred when saving configuration and some configuration may be lost.\r\nDo you want to leave this page?
-msgCheckCharset=** Notice **\n\nTo avoid miss any data unexpectedly, please confirm charactor columns' size before migration.\n\nBefore CUBRID 9.3, the length definition of a charactor column means how many bytes can be stored into the column, for example\: char(10) can store 10 single byte charactors or 5 double bytes charactors.\nSince CUBRID 9.3, the length means how many charactors can be stored into the column.\n\nDo you want to go to set up the columns length now?
+msgCheckCharset=** Notice **\n\nTo avoid miss any data unexpectedly, please confirm charactor columns' size before migration.\n\Less than CUBRID 9.3, the length definition of a charactor column means how many bytes can be stored into the column, for example\: char(10) can store 10 single byte charactors or 5 double bytes charactors.\nSince CUBRID 9.3, the length means how many charactors can be stored into the column.\n\nDo you want to go to set up the columns length now?
```

Comment:
The line break string 'n' is missing, so there is no line space.

Reply:
i'll add missing charactor in next commit